src/main/kotlin/platform/mcp/at/at-utils.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Minecraft Development for IntelliJ
+ *
+ * https://mcdev.io/
+ *
+ * Copyright (C) 2024 minecraft-dev
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, version 3.0 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.demonwav.mcdev.platform.mcp.at
+
+import com.demonwav.mcdev.facet.MinecraftFacet
+import com.demonwav.mcdev.platform.mcp.McpModuleType
+import com.demonwav.mcdev.platform.neoforge.NeoForgeModuleType
+import com.demonwav.mcdev.util.MinecraftVersions
+import com.demonwav.mcdev.util.SemanticVersion
+
+fun MinecraftFacet.usesSrgMemberNames(): Boolean {
+    if (!this.isOfType(NeoForgeModuleType)) {
+        return true
+    }
+
+    val mcpModule = this.getModuleOfType(McpModuleType) ?: return true
+    val mcVersion = mcpModule.getSettings().minecraftVersion?.let(SemanticVersion::tryParse) ?: return true
```

Comment:
These default `return` values seem suspicious to me. I don't know if it matters really but this might make more sense as a `TriState`, with the above options returning `UNKNOWN`. Or I guess just `null` with `Boolean?`.

Reply:
I agree, I used a simple Boolean to make it easier to use and have to make a decision anyway, and figured defaulting to using SRG is good enough. What would we do in case it is `UNKNOWN`?

Reply:
For this code that uses it, probably the same behavior. But it may be possible that later code will actually care, and doesn't really cost us anything to go ahead and do that.